From: Debian DNS Packaging <pkg-dns-devel@lists.alioth.debian.org>
Date: Fri, 24 Nov 2017 16:26:53 +0000
Subject: _multiarch

---
 isc-config.sh.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/isc-config.sh.in b/isc-config.sh.in
index a8a0a89..1e7903e 100644
--- a/isc-config.sh.in
+++ b/isc-config.sh.in
@@ -13,7 +13,6 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 exec_prefix_set=
 includedir=@includedir@
-libdir=@libdir@
 
 usage()
 {
@@ -132,7 +131,7 @@ if test x"$echo_libs" = x"true"; then
 	if test x"${exec_prefix_set}" = x"true"; then
 		libs="-L${exec_prefix}/lib"
 	else
-		libs="-L${libdir}"
+		libs=
 	fi
 	if test x"$libirs" = x"true" ; then
 		libs="$libs -lirs"
