From: Debian DNS Packaging <pkg-dns-devel@lists.alioth.debian.org>
Date: Fri, 24 Nov 2017 16:26:54 +0000
Subject: _ctxstart_no_sighandling

---
 lib/isc/unix/app.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/isc/unix/app.c b/lib/isc/unix/app.c
index 567f300..87d981d 100644
--- a/lib/isc/unix/app.c
+++ b/lib/isc/unix/app.c
@@ -262,6 +262,9 @@ isc__app_ctxstart(isc_appctx_t *ctx0) {
 	ctx->want_reload = false;
 	ctx->blocked = false;
 
+	if (!isc_bind9)
+		return (ISC_R_SUCCESS);
+
 #ifndef HAVE_SIGWAIT
 	/*
 	 * Install do-nothing handlers for SIGINT and SIGTERM.
