From: =?utf-8?b?T25kxZllaiBTdXLDvQ==?= <ondrej@debian.org>
Date: Tue, 21 May 2019 13:54:05 +0200
Subject: Use absolute srcdir path to protoc-c invocation

---
 lib/dns/Makefile.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/dns/Makefile.in b/lib/dns/Makefile.in
index 769b64c..19079e4 100644
--- a/lib/dns/Makefile.in
+++ b/lib/dns/Makefile.in
@@ -8,6 +8,7 @@
 # information regarding copyright ownership.
 
 srcdir =	@srcdir@
+abs_srcdir = 	@abs_srcdir@
 VPATH =		@srcdir@
 top_srcdir =	@top_srcdir@
 
@@ -227,7 +228,7 @@ ${OBJS}: include
 dnstap.@O@: dnstap.c dnstap.pb-c.c
 
 dnstap.pb-c.c dnstap.pb-c.h: dnstap.proto
-	$(PROTOC_C) --c_out=. --proto_path ${srcdir} dnstap.proto
+	$(PROTOC_C) --c_out=. --proto_path ${abs_srcdir} ${abs_srcdir}/dnstap.proto
 
 dnstap.pb-c.@O@: dnstap.pb-c.c
 
