diff -urNap lib/isccfg/Makefile lib/isccfg/Makefile
--- lib/isccfg/Makefile	2017-10-04 21:26:10.589804751 +0000
+++ lib/isccfg/Makefile	2017-10-04 21:26:55.969698162 +0000
@@ -506,6 +506,7 @@ libisccfg-export.la: ${OBJS}
 		-version-info ${LIBINTERFACE}:${LIBREVISION}:${LIBAGE} \
 		${OBJS} ${DNSLIBS} ${ISCLIBS}  \
 		${LIBS}
+	ln -sf .libs/libisccfg-export.so .
 
 timestamp: libisccfg-export.la
 	touch timestamp
